src/containers/Login/layouts/index.js

```diff
@@ -1,25 +1,21 @@
-import {Button, Scoll, StyleSheet, Text, View} from 'react-native';
-import {Input, Layout, PrimaryButton} from '../../../components';
+import {Layout, ModalLayout} from '../../../components';
+import React, {Fragment, useState} from 'react';
 
-import React from 'react';
-import styles from '../styles';
-
-const Login = ({doLogin, user}) => {
-  const userName = user?.additionalUserInfo?.profile?.given_name;
+import AuthContent from './authContent';
 
+const Login = () => {
+  const [islogin, setIsLogin] = useState(false);
   return (
-    <Layout backColor="DarkPrimary">
-      <View style={styles.container}>
-        <View style={styles.logoContainer}>
-          <Text>LOGO</Text>
-        </View>
-        <View style={styles.inputContainer}>
-          <Input placeholder="Usuario" />
-          <Input placeholder="Contraseña"  />
-          <PrimaryButton text="hello" />
-        </View>
-      </View>
-    </Layout>
+    <Fragment>
```

Comment:
at first, while I developed this part the recommendation before was applied but then I realize that if I render two components whenever I change this state the renders could interfere in the performance of the so I want to pass that state tho the auth component but I forgot that so I was rendering in the same way, the correct function will be corrected in the next commit